pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -46,6 +55,7 @@ type ReconcileJenkinsBaseConfiguration struct {
 
 // New create structure which takes care of base configuration
 func New(config configuration.Configuration, logger logr.Logger, jenkinsAPIConnectionSettings jenkinsclient.JenkinsAPIConnectionSettings) *ReconcileJenkinsBaseConfiguration {
+	verifyRouteAPI()
```

Comment:
is there a way to run it during the init phase instead ?

Reply:
Any other place you can suggest, I found this to be convenient as it is checked when the reconciler config is set which only happens once.